integration/bin/alluxio-worker-mesos.sh

```diff
@@ -16,8 +16,11 @@ source "${SCRIPT_DIR}/common.sh"
 ALLUXIO_WORKER_JAVA_OPTS="${ALLUXIO_WORKER_JAVA_OPTS:-${ALLUXIO_JAVA_OPTS}}"
 MESOS_LIBRARY_PATH="${MESOS_LIBRARY_PATH:-/usr/local/lib}"
 
-echo Mount ramdisk on worker
-${ALLUXIO_HOME}/bin/alluxio-mount.sh SudoMount
+# Try to mount ramdisk
+if [[ "${ALLUXIO_RAM_FOLDER}" == "/mnt/ramdisk" ]]; then
```

Comment:
this will not work if we ever want to start a worker with a ramdisk mounted in a different location or on OS X (for which the default is /Volumes/ramdisk)

I think that a better way to accomplish your goal would be to pass a parameter to the alluxio-worker-mesos.sh script that controls whether ramdisk should be mounted